usr/share/device-quirks/scripts/ayaneo/2021/2021.sh

```diff
@@ -13,16 +13,14 @@ fi
 PRODUCT_NAME="$(cat /sys/devices/virtual/dmi/id/product_name)"
 if [[ $USE_FIRMWARE_OVERRIDES == 1 ]]; then
   # Do DSDT override.
-  case $PRODUCT_NAME in
-    "AYANEO 2021")
-      echo "Enabling DSDT Override"
-      $DQ_PATH/scripts/override_dsdt "ayaneo_2021.dsl"
-      ;;
-    "AYANEO 2021 Pro Retro Power")
-      echo "Enabling DSDT Override"
-      $DQ_PATH/scripts/override_dsdt "ayaneo_2021_pro_retro_power.dsl"
-      ;;
-  esac
+  DSDT_OVERRIDES="ayaneo_2021_0x03.dsl ayaneo_2021_0xE3.dsl"
+  for dsdt in $DSDT_OVERRIDES; do
+    APPLY_PATCH=$($DQ_PATH/scripts/verify_dsdt $dsdt)
```

Comment:
Are we sure that we can rely on checksum alone for this? I think it's a good idea to do a DMI check AND a checksum check.

Reply:
The checksum should only match if the data is the same